baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/IllegalSafeLoggingArgument.java

```diff
@@ -90,13 +97,25 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
                             .setMessage(String.format(
                                     "Dangerous argument value: arg is '%s' but the parameter requires '%s'.",
                                     argumentSafety, parameterSafety))
+                            .addFix(getSuggestedFix(tree, state, argumentSafety))
```

Comment:
We can remove the "future work" item from the class-level javadoc describing this feature :-)